ddclient

```diff
@@ -329,24 +372,43 @@ my %variables = (
         'protocol'            => setv(T_PROTO, 0, 0, 1, 'dyndns2',            undef),
 
         'use'                 => setv(T_USE,   0, 0, 1, 'ip',                 undef),
+        'usev4'               => setv(T_USEV4, 0, 0, 1, undef,                undef),
+        'usev6'               => setv(T_USEV6, 0, 0, 1, undef,                undef),
         'ip'                  => setv(T_IP,    0, 0, 1, undef,                undef),
+        'ipv4'                => setv(T_IPV4,  0, 0, 1, undef,                undef),
+        'ipv6'                => setv(T_IPV6,  0, 0, 1, undef,                undef),
         'if'                  => setv(T_IF,    0, 0, 1, 'ppp0',               undef),
+        'ifv4'                => setv(T_IFV4,  0, 0, 1, undef,                undef),
+        'ifv6'                => setv(T_IFV4,  0, 0, 1, undef,                undef),
         'if-skip'             => setv(T_STRING,1, 0, 1, '',                   undef),
+#        'ifv4-skip'           => setv(T_STRING,1, 0, 1, '',                   undef),
+#        'ifv6-skip'           => setv(T_STRING,1, 0, 1, '',                   undef),
         'web'                 => setv(T_STRING,0, 0, 1, 'dyndns',             undef),
         'web-skip'            => setv(T_STRING,1, 0, 1, '',                   undef),
+        'webv4'               => setv(T_STRING,0, 0, 1, '',                   undef),
+        'webv4-skip'          => setv(T_STRING,1, 0, 1, '',                   undef),
+        'webv6'               => setv(T_STRING,0, 0, 1, '',                   undef),
+        'webv6-skip'          => setv(T_STRING,1, 0, 1, '',                   undef),
         'fw'                  => setv(T_ANY,   0, 0, 1, '',                   undef),
         'fw-skip'             => setv(T_STRING,1, 0, 1, '',                   undef),
+        'fwv4'                => setv(T_ANY,   0, 0, 1, '',                   undef),
+        'fwv4-skip'           => setv(T_STRING,1, 0, 1, '',                   undef),
+        'fwv6'                => setv(T_ANY,   0, 0, 1, '',                   undef),
+        'fwv6-skip'           => setv(T_STRING,1, 0, 1, '',                   undef),
         'fw-banlocal'         => setv(T_BOOL,  0, 0, 1, 0,                    undef),
         'fw-login'            => setv(T_LOGIN, 1, 0, 1, '',                   undef),
         'fw-password'         => setv(T_PASSWD,1, 0, 1, '',                   undef),
         'cmd'                 => setv(T_PROG,  0, 0, 1, '',                   undef),
         'cmd-skip'            => setv(T_STRING,1, 0, 1, '',                   undef),
+        'cmdv4'               => setv(T_PROG,  0, 0, 1, '',                   undef),
+#        'cmdv4-skip'          => setv(T_STRING,1, 0, 1, '',                   undef),
```

Comment:
Is there a reason you left some commented lines in here?

Reply:
Commented out settings represent those that are a "maybe". I put them in just for completeness pending final decision on whether they will be implemented or not.

Reply:
I expect them to be implemented eventually; they don't need to be implemented in this PR. If they will not be implemented in this PR then delete them, otherwise uncomment them.

Reply:
cleaned up now